GdkX11-4.0.gir

```diff
@@ -425,7 +425,7 @@ X servers that don't support the XCursor extension, all cursors
 may even fall back to a few default cursors.</doc>
         <return-value transfer-ownership="none">
           <doc xml:space="preserve">an Xlib Cursor.</doc>
-          <type name="xlib.Cursor" c:type="Cursor"/>
```

Comment:
These are all `XID`s, which are technically a `libc::c_ulong` and not pointers. `sizeof(unsigned long) != `sizeof(void*)` on some platforms (but I only know of Windows so...)

Reply:
Ah indeed, I should have checked what the XID always point to , updated